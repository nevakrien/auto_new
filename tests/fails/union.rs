use new_macro::new;

#[derive(new)]
union Dumby2{
	a:u8,
	b:u16
}