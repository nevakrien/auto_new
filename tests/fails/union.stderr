error: only structs are allowed with #[derive(new)]
 --> tests/fails/union.rs:4:7
  |
4 | union Dumby2{
  |       ^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/fails/union.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/fails/union.rs`
